src/drt/src/pa/FlexPA_prep.cpp

```diff
@@ -486,81 +529,43 @@ void FlexPA::genAPsFromRect(std::vector<std::unique_ptr<frAccessPoint>>& aps,
   }
 
   // gen all full/half grid coords
-  const int offset = is_macro_cell_pin ? hwidth : 0;
+  int offset = is_macro_cell_pin ? hwidth : 0;
+  offset = !is_macro_cell_pin || !use_center_line ? offset : 0;
```

Comment:
I think `!(is_macro_cell_pin && use_center_line) ` is simpler to read

Reply:
better even if switch the values and make it
`offset = (is_macro_cell_pin && use_center_line) ? 0 : offset`